### DISCUSSION TITLE
Unable to edit YAML in iOS app in 2021.10

### DISCUSSION THREAD

**Discussion by jordahlm** [home-assistant/frontend#10215] [Other feature requests]
After the upgrade to 2010.10.0, and the next day to 2010.10.2, I am no longer able to edit any YAML from the iOS app. When I try to enter any text, the cursor immediately jumps to the first line of whatever block is on the screen. I am able to enter 1 character at a time through tedious cursor moving, but any attempts to create a new line just result in a new line at the the line 1 position. I have tried resetting the frontend cache in the app and force-quitting the app, but the issue persists. This applies to any YAML that I have tried, including dashboards, cards, and automations

iOS version 14.7.1
App version 2021.8 (2021.216)

**Comment by bramkragten** [comment]
This should be an issue, not a discussion, but there is already an issue open for this atm (it is an upstream issue)
